packages/selfie-segmentation/android/build.gradle

```diff
@@ -12,7 +12,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:8.0.0'
+        classpath 'com.android.tools.build:gradle:8.0.2'
```

Comment:
Please revert this change so that we use the same Gradle version in all plugins. I will stay with `8.0.0` until the Capacitor team updates.